### DISCUSSION TITLE
What's status of JS version?

### DISCUSSION THREAD

**Discussion by vitonsky** [tursodatabase/turso#493] [Q&A]
Hi, that project looks potentially interesting, i looking for an embedded DB for my project that needed DB on client side.

I can't find info about WASM version of your project.

I see "JavaScript (wip)" but what it does mean? What's the roadmap for JS version and what's the current state? Also, do you have any docs for JS version? What's your plans about time ranges?

**Comment by penberg** [comment]
Hey @vitonsky, I moved this over to discussions. It's very early days for Limbo. You can find API coverage of the JavaScript package here: https://github.com/tursodatabase/limbo/blob/main/bindings/wasm/docs/api.md

**Reply by vitonsky** [reply]
Ok, what about your plans?
